init_qwat.sh

```diff
@@ -126,6 +126,7 @@ fi
 # Create extenstions
 psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c "CREATE EXTENSION IF NOT EXISTS hstore;"
 psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c "CREATE EXTENSION IF NOT EXISTS postgis;"
+psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto";'
```

Comment:
@3nids This statement is missing in the delta file and will result in a sql failure when updating, could you please add it to the delta file?

Reply:
done